go/login/login_test.go

```diff
@@ -0,0 +1,181 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package login
+
+import (
+	"encoding/hex"
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/google/glome/go/glome"
+)
+
+const (
+	tvsCnt = 2
+)
```

Comment:
Can't you just use `len()` instead of defining a constant here? `[][]typ` is a valid type (slice of slices).

Reply:
Fixed.